conf/test_ancient_dna.config

```diff
@@ -33,7 +33,7 @@ params {
     skip_krona                           = true
     min_length_unbinned_contigs          = 1
     max_unbinned_contigs                 = 2
-    busco_db                             = "https://busco-data.ezlab.org/v5/data/lineages/bacteria_odb10.2024-01-08.tar.gz"
+    busco_db                             = "https://busco-data.ezlab.org/v5/data/lineages/bacteria_odb12.2024-11-14.tar.gz"
```

Comment:
This file doesn't exist any more. Latest version is

https://busco-data.ezlab.org/v5/data/lineages/bacteria_odb12.2025-04-11.tar.gz

Reply:
It might be worth bundling a busco lineage if they're small enough to avoid the problem of file name changes?

Reply:
Current bacteria_odb12 is over 50MB, so I will leave bacteria_odb10 for tests instead.
[bacteria_odb10.2024-01-08.tar.gz](https://busco-data.ezlab.org/v5/data/lineages/bacteria_odb10.2024-01-08.tar.gz)                   04-Mar-2025 01:01      6M
[bacteria_odb12.2025-04-11.tar.gz](https://busco-data.ezlab.org/v5/data/lineages/bacteria_odb12.2025-04-11.tar.gz)                   11-Apr-2025 20:43     54M